app/src/main/java/com/github/studydistractor/sdp/maps/MapsActivity.kt

```diff
@@ -83,7 +88,7 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
         }
         marker?.tag = activity
         map?.setOnInfoWindowClickListener { activityMarker ->
-            val procActivity = activityMarker.tag as ProcrastinationActivity
+            val procActivity = activityMarker.tag as Distraction
```

Comment:
We can change the name of this variable to `distractionActivity` to be consistent with the renaming.